README.md

```diff
@@ -30,6 +32,47 @@ pip install -r requirements.txt
 npm install
 ```
 
+### Autoenv setup
+
+The autoenv tool manages environment variables for the project and
+automatically activates the virtual environment when you enter the project
+directory. Install it *outside* the virtual environment.
+
+```
+deactivate
+pip install autoenv==1.0.0
+```
+
+Then, add ``source `which activate.sh` `` to your .bashrc or other startup
+file, and rerun that file. Now, when you leave and reenter the directory, the
+virtual environment activates automatically. 
+
+### Database setup
+
+The database is managed by Alembic. To create it for the first time, first
+enter the project directory to activate the virtual environment. Then, initialize
+the migrations directory, set up a migration, and upgrade to it.
+```
+python manage.py db init
+python manage.py db migrate
```

Comment:
Remover db init and db migrate, as they only need to be run when starting a project (or in the case of migrate, when updating models). Because the migrations are already in git, we only need to run upgrade to get up to speed here.